clouddriver-kubernetes/src/integration/java/com/netflix/spinnaker/clouddriver/kubernetes/it/DeployManifestIT.java

```diff
@@ -1351,6 +1349,80 @@ public void shouldDeployBlueGreenReplicaSet() throws IOException, InterruptedExc
         1, podNames.size(), "Only one pod expected to have the label for traffic selection");
   }
 
+  @DisplayName(
+      ".\n===\n"
+          + "Given a deployment yaml with red/black deployment traffic strategy\n"
+          + "  and an existing service\n"
+          + "When sending deploy manifest request two times\n"
+          + "  And sending disable manifest one time\n"
+          + "Then there are two deployments with only the last one receiving traffic\n===")
+  @Test
+  public void shouldDeployRedBlackDeploymentKind() throws IOException, InterruptedException {
```

Comment:
let's also add a test for the new `blue-green` traffic management strategy

Reply:
addressed